pkg/index/mysql/mysql.go

```diff
@@ -0,0 +1,106 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/go-sql-driver/mysql"
+	"github.com/gomods/athens/pkg/config"
+	"github.com/gomods/athens/pkg/errors"
+	"github.com/gomods/athens/pkg/index"
+	_ "github.com/lib/pq"
+)
+
+func New(cfg *config.MySQL) (index.Indexer, error) {
+	dataSource := getMySQLSource(cfg)
+	db, err := sql.Open("mysql", dataSource)
+	if err != nil {
+		return nil, err
+	}
+	if err = db.Ping(); err != nil {
+		return nil, err
+	}
+	_, err = db.Exec(mysqlTable)
+	if err != nil {
+		return nil, err
+	}
+	return &indexer{db}, nil
+}
+
+const mysqlTable = `
```

Comment:
nit: you called this `schema` in the postgres driver, might be helpful to keep the terminology the same between the two implementations

Reply:
Yep!